_unittest/test_26_emit.py

```diff
@@ -23,5 +29,46 @@ def test_objects(self):
         assert self.aedtapp.solution_type
         assert isinstance(self.aedtapp.existing_analysis_setups, list)
         assert isinstance(self.aedtapp.setup_names, list)
+        assert isinstance(self.aedtapp.modeler.components, EmitComponents)
         assert self.aedtapp.modeler
         assert self.aedtapp.oanalysis is None
+
+    def test_create_components(self):
+        radio = self.aedtapp.modeler.components.create_component(
+            "New Radio", "TestRadio")
+        assert radio.name == "TestRadio"
+        assert isinstance(radio, EmitComponent)
+        antenna = self.aedtapp.modeler.components.create_component(
+            "Antenna", "TestAntenna")
+        assert antenna.name == "TestAntenna"
+        assert isinstance(antenna, EmitComponent)
+
+    @pytest.mark.skipif(config["desktopVersion"] < "2021.2",
```

Comment:
@myoung301 @maxcapodi78  
1. What would happen if AEDT install is a Patched install - isn't the version in that case, for e.g., 2021.2 **.1**?
2. It seems a number or string is being compared to (less than) a string, will that work? I guess I might be missing something, does the pytest.mark.skipif decorator do some kind of conversion of the string to fractional/number?

Reply:
you have always to consider "2021.2" the patch version is only managed when starting the desktop since there is no real difference between 2021.2 and 2021.2.1